.circleci/config.yml

```diff
@@ -56,16 +56,16 @@ commands:
       - restore_cache:
           name: Restore lotus artefacts cache
           keys:
```

Comment:
Every time APIs change on Lotus, we have to update the commit that our CI points to, so that the `devnet`, `boost` and `lotus` are compatible.